pkg/cfn/builder/cluster_test.go

```diff
@@ -360,6 +361,46 @@ var _ = Describe("Cluster Template Builder", func() {
 				Expect(clusterTemplate.Resources).ToNot(HaveKey("PublicSubnetRoute"))
 				Expect(clusterTemplate.Resources).To(HaveKey(ContainSubstring("PrivateRouteTable")))
 			})
+			When("ipv6 cluster is enabled", func() {
+				BeforeEach(func() {
+					cfg.VPC.Network.IPFamily = api.IPV6Family
+				})
+				AfterEach(func() {
+					cfg.VPC.Network.IPFamily = api.IPV4Family
+				})
```

Comment:
![image](https://user-images.githubusercontent.com/19669095/140720094-d6fb627e-e297-4955-9958-0afd500935b1.png)

is the cfg not recreated after each test? seems like a big smell that we would need this :thinking:

Reply:
It was probably some left in precaution I had :D